testSetupFile.js

```diff
@@ -8,3 +8,6 @@
 // Some of the `jest-runtime` tests are very slow and cause
 // timeouts on travis
 jest.setTimeout(70000);
+
+// this module does some funky stuff with `require.cache`, flooding the terminal with output
+jest.mock('stealthy-require', () => (_, m) => m());
```

Comment:
```sh-session
$ yarn why stealthy-require
=> Found "stealthy-require@1.1.1"
info Reasons this module exists
   - "_project_#jest-environment-jsdom#jsdom#request-promise-native" depends on it
   - Hoisted from "_project_#jest-environment-jsdom#jsdom#request-promise-native#stealthy-require"
```

https://github.com/jsdom/jsdom/issues/2792

Reply:
@SimenB  should it be a best practice recommended for everyone?